jekyll/_cci2/container-runner.adoc

```diff
@@ -145,6 +145,33 @@ agent:
           - name: "other"
 ```
 
+[#custom-secret]
+=== Custom Token Secret
+
+Using the above configuration will provision a Kubernetes secret containing your Resource Class tokens. In some circumstances you may wish to provision your own secret, or you simply might not want to specify the tokens via helm. Instead you can provision your own Kubernetes secret containing your tokens and specify its name in the `agent.customSecret` field.
+
+The secret should contain a field for each Resource Class, using the Resource Class name as the key and the token as the value. Consider the following `resourceClasses` configuration:
+
+```yaml
+agent:
+  resourceClasses:  
+    circleci-runner/resourceClass:
+      metadata:
+        annotations:
+          custom.io: my-annotation
+    
+    circleci-runner/resourceClass2:
```

Comment:
For values that the reader needs to substitute for their own data, we use `<my-token>`, <my-resource-class> or whatever is appropriate. Could you update this snippet to use this format?

Reply:
Oh and the one below to match